src/status_im2/contexts/quo_preview/common.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im2.contexts.quo-preview.common
+  (:require
+    [quo2.core :as quo]
+    [quo2.theme :as theme]
+    [utils.re-frame :as rf]))
+
+
+(defn navigation-bar
+  []
+  (let [logged-in?    (rf/sub [:multiaccount/logged-in?])
+        has-profiles? (boolean (rf/sub [:profile/profiles-overview]))
+        root          (if has-profiles? :profiles :intro)
+        light?        (= (theme/get-theme) :light)]
+    [quo/page-nav
+     {:type       :title
+      :title      "Quo2 components preview"
```

Comment:
It would be nice if we could display the component name in the header when the component preview page is open.

Reply:
Nice suggestion, but out of scope of this PR

Reply:
Got it. I will add it to the Preview suggestions if it's not already there.

Reply:
If I'm not mistaken somebody else suggested this idea too 👍🏼